### PR TITLE
#20: Fixed bug with non-existent dir when creating new entity, renamed stub configs, added namespace for DatabaseSeeder

### PR DESCRIPTION
**Pull request by vitgrams**
for issue: #20

### CODE REVIEW

**Comment by vitgrams** [comment]
@DenTray Done. Could you please review this code?

File: src/Generators/FactoryGenerator.php [hunk 1]
```diff
@@ -83,18 +83,16 @@ protected function generateToGenericClass(): string
 
     public function generate()
     {
-        if (floatval(app()->version()) >= 8) {
-            $createMessage = $this->generateSeparateClass();
-        } else {
-            $createMessage = $this->generateToGenericClass();
-        }
+        $createMessage = (floatval(app()->version()) >= 8)
```
Comment (DenTray) [inline src/Generators/FactoryGenerator.php:86]: could we use https://www.php.net/manual/en/function.version-compare.php ?

**Review by DenTray — CHANGES_REQUESTED** [review]

**Comment by vitgrams** [comment]
@DenTray Fixed code review remarks.

File: src/Generators/SeederGenerator.php [hunk 2]
```diff
@@ -24,14 +24,21 @@ public function generate()
         $this->checkConfigs();
 
         if (!file_exists($this->databaseSeederPath)) {
+            $databaseSeederPathSegments = explode('/', $this->databaseSeederPath);
+            array_pop($databaseSeederPathSegments);
+            $databaseSeederDir = implode('/', $databaseSeederPathSegments);
```
Comment (DenTray) [inline src/Generators/SeederGenerator.php:29]: please use helper for it https://github.com/RonasIT/laravel-helpers/blob/master/src/helpers.php#L395

**Review by DenTray — CHANGES_REQUESTED** [review]

**Comment by vitgrams** [comment]
@DenTray Fixed review remarks.

File: src/Generators/SeederGenerator.php [hunk 3]
```diff
@@ -24,14 +24,19 @@ public function generate()
         $this->checkConfigs();
 
         if (!file_exists($this->databaseSeederPath)) {
+            list(, $databaseSeederDir) = extract_last_part($this->databaseSeederPath, '/');
```
Comment (DenTray) [inline src/Generators/SeederGenerator.php:27]:
```suggestion
            list($basePath, $databaseSeederDir) = extract_last_part($this->databaseSeederPath, '/');
```

**Review by DenTray — APPROVED** [review]

**Pull request merged**